actix-web-codegen/src/lib.rs

```diff
@@ -1,118 +1,109 @@
 #![recursion_limit = "512"]
+//! Actix-web codegen module
+//!
+//! Generators for routes and scopes
+//!
+//! ## Route
+//!
+//! Macros:
+//!
+//! - [handler](attr.handler.html)
+//! - [get](attr.get.html)
+//! - [post](attr.post.html)
+//! - [put](attr.put.html)
+//! - [delete](attr.delete.html)
+//!
+//! ### Attributes:
+//!
+//! - `"path"` - Raw literal string with path for which to register handle. Mandatory.
+//! - `async` - Attribute to indicate that registered function is asynchronous.
```

Comment:
remove this too

Reply:
Ehm... I think it might be useful to specify when you cannot auto-detect asyncness... Are you sure?

Reply:
Yes

Reply:
Done